### DISCUSSION TITLE
summary variable

### DISCUSSION THREAD

**Discussion by zeecode07** [pybamm-team/PyBaMM#3203] [Q&A]
what is the syntax for displaying one of these summary_variables? e.g. want to display only LLI graphs @martinjrobins @tinosulzer @brosaplanella 
![image](https://github.com/pybamm-team/PyBaMM/assets/137913115/11f98abf-53e0-4474-9ba0-b04ad37c3fdb)

**Comment by valentinsulzer** [comment]
You can pass in `output_variables` as a list to `plot_summary_variables` to overwrite the default ones. Or take a look at the source code if you want to write your own version: https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/plotting/plot_summary_variables.py

By the way, doing `?pybamm.plot_summary_variables` will give you the function's docstring in jupyter, which can be helpful for finding out answers to this kind of question
